lib/shopify_cli/theme/development_theme.rb

```diff
@@ -9,6 +9,8 @@ module Theme
     API_NAME_LIMIT = 50
 
     class DevelopmentTheme < Theme
+      attr_accessor :created_at_runtime
```

Comment:
Curious – since we have a wrapper method `created_at_runtime?`, is this accessor needed?